sdk/communication/azure-communication-callautomation/src/main/java/com/azure/communication/callautomation/models/DtmfTone.java

```diff
@@ -68,8 +68,72 @@ public static DtmfTone fromString(String name) {
         return fromString(name, DtmfTone.class);
     }
 
+    /** @return known Tone values in char. */
+    public char toChar() {
+        return convertToChar();
```

Comment:
what is the intention of having just a one line method? would it be simpler to add the contents of `convertToChar` in this method?

Reply:
will remove the private embed function